config/owasp/suppressions.xml

```diff
@@ -63,13 +63,8 @@
     <cve>CVE-2018-1260</cve>
   </suppress>
   <suppress>
-    <notes>
-      <![CDATA[
-        file name: jackson-databind-2.9.8.jar
-        reason for exclusion: risk does not exist because one of preconditions is not met - mysql-connector-java library is not used
-      ]]>
-    </notes>
+    <notes><![CDATA[file name: jackson-databind-2.9.9.jar]]></notes>
```

Comment:
A `2.9.9.1` is supported to fix it according to https://github.com/FasterXML/jackson-databind/blob/jackson-databind-2.9.9.1/release-notes/VERSION-2.x.

Have you tried to bring that version implicitly without excluding older version?